src/integration-test/java/com/mongodb/jdbc/integration/DCIntegrationTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2024-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.jdbc.integration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.mongodb.jdbc.MongoConnection;
+import com.mongodb.jdbc.Pair;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class DCIntegrationTest {
+
+    /** Tests that the driver can work with SRV-style URIs. */
+    @Test
+    public void testConnectWithSRVURI() throws SQLException {
+        String mongoHost = System.getenv("SRV_TEST_HOST");
+        assertNotNull(mongoHost, "SRV_TEST_HOST variable not set in environment");
+        String mongoURI =
+                "mongodb+srv://"
+                        + mongoHost
+                        + "/?readPreference=secondaryPreferred&connectTimeoutMS=300000";
+        String fullURI = "jdbc:" + mongoURI;
+
+        String user = System.getenv("SRV_TEST_USER");
+        assertNotNull(user, "SRV_TEST_USER variable not set in environment");
+        String pwd = System.getenv("SRV_TEST_PWD");
+        assertNotNull(pwd, "SRV_TEST_PWD variable not set in environment");
+        String authSource = System.getenv("SRV_TEST_AUTH_DB");
+        assertNotNull(authSource, "SRV_TEST_AUTH_DB variable not set in environment");
+
+        Properties p = new java.util.Properties();
+        p.setProperty("user", user);
+        p.setProperty("password", pwd);
+        p.setProperty("authSource", authSource);
+        p.setProperty("database", "test");
+
+        MongoConnection conn = (MongoConnection) DriverManager.getConnection(fullURI, p);
```

Comment:
This is now failing because the library is not loaded. That seems like a good sign! I guess I should set it up so that these tests are run with the library built and in the appropriate place? Or should we defer that to a follow-up ticket and have this test continue to expect a `SQLException`?

Reply:
I added a [sample](https://github.com/mongodb/mongo-jdbc-driver/tree/master/src/test/resources/MongoSqlLibraryTest) library that you can [move](https://github.com/mongodb/mongo-jdbc-driver/blob/904969e718ee013df4d493f2e8737eedea58536a/.evg.yml#L374) to the `build/classes/java/` directory that should get this past the library loading error.  If that works successfully then that should be good since we can then test the buildInfo logic here.